### PR TITLE
fix: fix card image

### PR DESCRIPTION
**Pull request by LouiaR**
Type: fix

Issue: the image  is not displayed on Card component

Screen Shot 2019-08-30 at 14 10 37

### CODE REVIEW

File: src/components/Molecules/Card/Card.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-Card
+# Card
 
 ```js
 const data = require('../../../styleguide/data/data').default;
 import Text from '../../Atoms/Text/Text';
 import Link from '../../Atoms/Link/Link';
 
-<Card image={data.image} images={data.image} backgroundColor="yellow">
+<Card imageLow={data.image} images={data.image} backgroundColor="yellow">
```
Comment (gusliedke) [inline src/components/Molecules/Card/Card.md]: We need add `images` ie. `images={data.images}`

Comment (LouiaR) [reply src/components/Molecules/Card/Card.md]: Yeah It's the props name expected by `Picture` atom

Comment (gusliedke) [reply src/components/Molecules/Card/Card.md]: I mean in `images={data.image}` you need to add `s` ie `image={data.images}`

Comment (LouiaR) [reply src/components/Molecules/Card/Card.md]: Sure

**Pull request merged**